indexer/indexer.go

```diff
@@ -159,16 +142,32 @@ func NewIndexer(cfg Config, gitVersion string) (*Indexer, error) {
 		return nil, err
 	}
 
+	var addrManager services.AddressManager
+	if cfg.Bedrock {
+		addrManager, err = services.NewBedrockAddresses(
```

Comment:
Same as above, I don't like tying so much to bedrock/legacy

Is it annoying to make it just not index the bedrock contracts if we don't pass in the config for them for now and then post launch just have it always index them?

Reply:
We will always need to index the legacy stuff because the legacy withdrawals will always be a thing and for a consistent ux, doing something like `GET /withdrawals/{user}` should show all historical withdrawals

Reply:
If we remove indexing ability of the old system, we will never be able to sync from scratch again?

I see your point, just worry about creating future work for ourselves now. I'm on board with leaving it as is